### DISCUSSION TITLE
If there is no jailbroken device, how can I obtain a decrypted IPA

### DISCUSSION THREAD

**Discussion by makisekurisu-jp** [arichornlover/uYouEnhanced#497] [Q&A]
I only have a Windows computer, no jailbroken Apple devices. How can I obtain a decrypted IPA?

**Comment by jianmingyong** [comment]
> I only have a Windows computer, no jailbroken Apple devices. How can I obtain a decrypted IPA?

Google is your best friend. I am unable to share a direct link that contains it but you could find them by googling the correct terms.

**Comment by makisekurisu-jp** [comment]
> > I only have a Windows computer, no jailbroken Apple devices. How can I obtain a decrypted IPA?
> 
> 
> 
> Google is your best friend. I am unable to share a direct link that contains it but you could find them by googling the correct terms.

Thank you for your answer. I have found it, but forgot to close this issue.
